doc/manuals/admin/yum.md

```diff
@@ -0,0 +1,49 @@
+## Using YUM repositories
+
+There are 2 available packages:
+- nightly 
+- release
+ 
+You can manually add config for yum repo or download it from FIWARE public repository.
+
+Use this config for nightly packages:
+```
+[fiware-nightly]
+name=FIWARE nightly repository
+baseurl=http://nexus.lab.fiware.org/repository/el/$releasever/$basearch/nightly
+enabled=1
+gpgcheck=0
+priority=1
+
+```
+or download it from [FIWARE public repository](http://nexus.lab.fiware.org/repository/raw/public/repositories/el/7/x86_64/fiware-nightly.repo)
+```
+curl -o /etc/yum.repos.d/fiware-nightly.repo http://nexus.lab.fiware.org/repository/raw/public/repositories/el/7/x86_64/fiware-nightly.repo
+
+```
+
+Use this config for release packages:
+```
+[fiware-release]
+name=FIWARE release repository
+baseurl=http://nexus.lab.fiware.org/repository/el/$releasever/$basearch/release
+enabled=1
+gpgcheck=0
+priority=1
+
+```
+or download it from [FIWARE public repository](http://nexus.lab.fiware.org/repository/raw/public/repositories/el/7/x86_64/fiware-release.repo)
+```
+curl -o /etc/yum.repos.d/fiware-release.repo http://nexus.lab.fiware.org/repository/raw/public/repositories/el/7/x86_64/fiware-release.repo
+```
+
+Keep in mind, configs from FIWARE public repository have https connection and this can cause problems, to avoid them see [SSL configuration](#SSL)
+
+## SSL configuration
+If you want to use https versions of yum repositories, you have to add [GeoTrust](http://nexus.lab.fiware.org/repository/raw/public/ca/geotrust.pem) root certificates to your system.
+
+Download and add certificate
+```
+curl -o /etc/pki/ca-trust/source/anchors/geotrust.pem http://nexus.lab.fiware.org/repository/raw/public/ca/geotrust.pem 
```

Comment:
What about if the user has previously a /etc/pki/ca-trust/source/anchors/geotrust.pem exists? Could it happen? Does the concern makes sense?

(My question is motivated becuase I don't know which kind of stuff is at /etc/pki/ca-trust/source/anchors)

Reply:
Nothing will happen, not even a warning

Reply:
So you mean that the geotrust.pem download from fiware.org could override an existing geotrust.pem file (with the potential breakage in existing applñication using that .pem in the users system)?

Reply:
Small bug was found, fixed, and this block is not needed anymore

Reply:
fixed in ce41e406315343b6c61358ad41eda2854b4a1b63